apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/pathAssertions.kt

```diff
@@ -156,6 +156,9 @@ val <T : Path> Expect<T>.parent: Expect<Path>
 fun <T : Path> Expect<T>.parent(assertionCreator: Expect<Path>.() -> Unit): Expect<T> =
     ExpectImpl.path.parent(this).addToInitial(assertionCreator)
 
+//TODO to be completed.
+fun <T : Path> Expect<T>.resolve(assertionCreator: Expect<Path>.() -> Unit, other: String): Expect<T> =
```

Comment:
switch arguments, assertionCreator-lambda should always be the last argument
```suggestion
fun <T : Path> Expect<T>.resolve(other: String, assertionCreator: Expect<Path>.() -> Unit): Expect<T> =
```